sklearn/model_selection/_split.py

```diff
@@ -543,11 +543,11 @@ class StratifiedKFold(_BaseKFold):
     >>> from sklearn.model_selection import StratifiedKFold
     >>> X = np.array([[1, 2], [3, 4], [1, 2], [3, 4]])
     >>> y = np.array([0, 0, 1, 1])
-    >>> skf = StratifiedKFold(n_splits=2)
+    >>> skf = StratifiedKFold(n_splits=2, random_state=0)
```

Comment:
I don't understand how your comment relates to my comment. You said the tests failed. You could have fixed that with a ``...``.
If we want the ``repr`` to be nicer if ``shuffle`` is not set, we could not change it then.
I don't think we should change the repr in other cases and hide information from the user.